src/app/views/assignments/models/searchQuestionVM.ts

```diff
@@ -1,8 +1,4 @@
-export class SearchQuestionPaperVM {
-  constructor(pageNumber = 1, pageSize = 25) {
-    this.pageNumber = pageNumber;
-    this.pageSize = pageSize;
-  }
-  pageNumber: number = 1;
-  pageSize: number = 25;
-}
+export class SearchQuestionPaperVM{
+    pageNumber : number = 1;
+    pageSize : number = 25;
```

Comment:
We need to have paginated implemented along with sort column and all. I believe there were more changes here which is removed.

{
  "nameRegexPattern": "string",
  "next": true,
  "pageNumber": 0,
  "pageSize": 0,
  "questionPaperId": "string",
  "sortColumn": "string",
  "sortOrder": "string",
  "status": "string",
  "subject": "string",
  "tags": [
    "string"
  ],
  "updateEndTime": "string",
  "updateStartTime": "string"
}

based on /api/v1/test/config/search